src/app/static/src/app/components/adr/SelectRelease.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+    <div id="selectRelease" v-if="newDatasetId && releases.length">
+        <div class="pt-1">
```

Comment:
On my screen this is quite cramped 
```suggestion
        <div class="pt-2">
```